### PR TITLE
Massive rename operation

### PR DESCRIPTION
**Pull request by joemccall86**
DO NOT MERGE. This is just to evaluate how GitHub will display a massive
rename operation, and to see if it is really feasible to use for
code-review.

### CODE REVIEW

**Review by matt-nicholson-intrepid — COMMENTED** [review]
Leaving a test comment about PR

**Review by matt-nicholson-intrepid — CHANGES_REQUESTED** [review]
I'm not sure if I need permissions to comment on a specific line, or if it's not a feature of GitHub

File: classes/production/grails-bootstrap/grails.build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+grails.version=3.3.0.BUILD-SNAPSHOT
```
Comment (matt-nicholson-intrepid) [inline classes/production/grails-bootstrap/grails.build.properties:1]: I see the button now, was looking at the wrong margin

Comment (joemccall86) [reply classes/production/grails-bootstrap/grails.build.properties:1]: It appears that syntax formatting **should** be obvious, given the UI buttons on the comment input box.

**Pull request closed without merging**